build/templates/UmbracoProject/UmbracoProject.csproj

```diff
@@ -21,12 +22,8 @@
     </ItemGroup>
 
     <ItemGroup>
-        <Content Include="App_Plugins/**" CopyToOutputDirectory="Always" />
-        <Content Include="umbraco/**" CopyToOutputDirectory="Always" />
-        <Content Remove="umbraco/Data/**" />
-        <Content Remove="umbraco/Logs/**" />
-        <Content Remove="umbraco/mediacache/**" />
-        <Content Remove="umbraco/models/**" />
+        <Content Include="App_Plugins/**" CopyToOutputDirectory="Always" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);@(_ContentIncludedByDefault)" />
+        <Content Include="umbraco/**" CopyToOutputDirectory="Always" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);@(_ContentIncludedByDefault)" />
         <Compile Include="umbraco/models/**" Exclude="**/*.flag" />
```

Comment:
ood.flag makes its way into Umbraco.StaticAssets, investigate (this might just be when making the nupkg locally)